src/components/UserSelector.tsx

```diff
@@ -73,7 +56,7 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
```

Comment:
It's not recommended to use `href` with a hash value in this context. This will change the URL and can lead to unexpected behavior. Instead, consider using a `button` for this purpose. This will also improve accessibility, as buttons are expected to have an interactive role, while links are expected to navigate to a different resource or area of the same page.